### PR TITLE
docs: update mirror link

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
https://github.com/ant-design/ant-design/issues/35090

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Add antgroup mirror link          |
| 🇨🇳 Chinese | 增加 antgroup 镜像链接          |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
内网用户弹个窗。

**Comment by afc163** [comment]
图片

加个 icon 好看一点。

File: site/theme/template/Layout/Header/index.tsx [hunk 1]
```diff
@@ -109,6 +109,20 @@ class Header extends React.Component<HeaderProps, HeaderState> {
...
+        ) {
+          Modal.info({
+            title: '提示',
+            content: '国内镜像已经上线，推荐访问国内镜像以获得更佳体验～',
+            okText: '前往',
+            onOk: () => {
+              window.open('https://ant-design.antgroup.com', '_self');
+            },
+            closable: true,
+          });
```
Comment (afc163) [inline site/theme/template/Layout/Header/index.tsx:124]: 1. 识别只有内网用户弹，文案也相应修改
2. 记 localStorage，点过『前往』或者『不再弹出』后不再出现。

**Comment by MadCcc** [comment]
> 加个 icon 好看一点。

好像没有 gitee 的 icon

File: site/theme/template/Layout/Header/index.tsx [hunk 2]
```diff
@@ -109,6 +109,20 @@ class Header extends React.Component<HeaderProps, HeaderState> {
         this.setState({
           showTechUIButton: true,
         });
+        if (
+          process.env.NODE_ENV === 'production' &&
+          !window.location.href.includes('ant-design.antgroup.com')
+        ) {
+          Modal.info({
+            title: '提示',
+            content: '国内镜像已经上线，推荐访问国内镜像以获得更佳体验～',
```
Comment (afc163) [inline site/theme/template/Layout/Header/index.tsx:118]:
```suggestion
            content: '内网用户推荐访问国内镜像以获得极速体验～🚀',
```

**Comment by afc163** [comment]
https://gitee.com/ 官网抠一个。

**Comment by afc163** [comment]
https://www.iconfont.cn/search/index?searchType=icon&q=gitee

File: site/theme/template/Layout/Header/index.tsx (hunk 1 above)
Comment (MadCcc) [reply site/theme/template/Layout/Header/index.tsx:124]: > 1. 识别只有内网用户弹

这里是和 TechUI 相同的判断

Comment (MadCcc) [reply site/theme/template/Layout/Header/index.tsx:139]: done

File: site/theme/template/Layout/Header/index.tsx (hunk 2 above)
Comment (MadCcc) [reply site/theme/template/Layout/Header/index.tsx]: done

File: site/theme/template/Layout/Header/index.tsx (hunk 1 above)
Comment (afc163) [reply site/theme/template/Layout/Header/index.tsx:139]: 图片

只有一个按钮了。

File: site/theme/template/Layout/Header/index.tsx [hunk 3]
```diff
@@ -109,6 +119,25 @@ class Header extends React.Component<HeaderProps, HeaderState> {
...
+            content: '内网用户推荐访问国内镜像以获得极速体验～🚀',
+            okText: '前往',
+            onOk: () => {
+              window.open('https://ant-design.antgroup.com', '_self');
+              disableAntdMirrorModal();
+            },
+            cancelText: '不再弹出',
+            onCancel: () => {
+              disableAntdMirrorModal();
+            },
```
Comment (afc163) [inline site/theme/template/Layout/Header/index.tsx:138]: closable 可以加。

File: site/theme/template/Layout/Header/index.tsx [hunk 4]
```diff
@@ -109,6 +119,25 @@ class Header extends React.Component<HeaderProps, HeaderState> {
...
         });
+        if (
+          process.env.NODE_ENV === 'production' &&
+          !window.location.href.includes('ant-design.antgroup.com') &&
+          shouldOpenAntdMirrorModal()
+        ) {
+          Modal.info({
+            title: '提示',
+            content: '内网用户推荐访问国内镜像以获得极速体验～🚀',
+            okText: '前往',
```
Comment (afc163) [inline site/theme/template/Layout/Header/index.tsx:130]:
```suggestion
            okText: '立刻前往',
```

File: site/theme/template/Layout/Header/index.tsx [hunk 5]
```diff
@@ -109,6 +119,25 @@ class Header extends React.Component<HeaderProps, HeaderState> {
...
           showTechUIButton: true,
         });
+        if (
+          process.env.NODE_ENV === 'production' &&
+          !window.location.href.includes('ant-design.antgroup.com') &&
+          shouldOpenAntdMirrorModal()
+        ) {
+          Modal.info({
+            title: '提示',
+            content: '内网用户推荐访问国内镜像以获得极速体验～🚀',
```
Comment (afc163) [inline site/theme/template/Layout/Header/index.tsx:129]:
```suggestion
            content: '内网用户推荐访问国内镜像以获得极速体验～',
```

**Pull request merged**